exercises/decimal/example.rs

```diff
@@ -56,10 +56,7 @@ impl Decimal {
         }
         Some(Decimal::new(
             try_opt!(digits.parse::<BigInt>().ok()),
```

Comment:
Might be out of scope of this PR, but I suspect that we could also simplify this line:

```suggestion
            digits.parse().ok()?,
```